packages/Algernon.package/ALGResultItemMorph.class/instance/morphLayoutFractions.st

```diff
@@ -7,7 +7,7 @@ morphLayoutFractions
 	| actionEnd |
 	actionEnd := (self isTitle not
 					and: [self item algerType hasChildren])
-				ifTrue: [0.94]
```

Comment:
I don't really know what this change is about, but if you say that it's helpful it's ok😅

Reply:
It's graphical stuff. It looks good so it's correct! 😆